pkg/api/graph/types.go

```diff
@@ -222,8 +223,14 @@ func ImageStreamTag(g MutableUniqueGraph, namespace, name, tag string) graph.Nod
 	if len(tag) == 0 {
 		tag = "latest"
 	}
+	var uname UniqueName
+	if strings.Index(name, ":") == -1 && strings.Index(name, "@") == -1 {
```

Comment:
right..  basically "if this doesn't contain a :tag or @id" suffix, it's an ImageStream and we need to append the "tag" value from the Tag field (or latest if the Tag field is empty)"

otherwise, it's an ImageStreamTag or ImageStreamImage and we can use the name as is.